vdr/cmd/cmd_test.go

```diff
@@ -240,6 +241,72 @@ func TestEngine_Command(t *testing.T) {
 			assert.Contains(t, buf.String(), "invalid")
 		})
 	})
+
+	t.Run("deactivate", func(t *testing.T) {
+		t.Run("ok", func(t *testing.T) {
+			cmd := Cmd()
+			s := httptest.NewServer(http2.Handler{StatusCode: http.StatusOK})
+			os.Setenv("NUTS_ADDRESS", s.URL)
+			defer os.Unsetenv("NUTS_ADDRESS")
+			core.NewClientConfig().Load(cmd.Flags())
+			defer s.Close()
+
+			inBuf := new(bytes.Buffer)
```

Comment:
lots of duplicate code for setting up the buffer. Could be placed in a func?

Reply:
Fixed 👍